persistence/sql/migrations/sql/20221009000000000000_add_session_privileged_until_column.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "sessions" ADD COLUMN "privileged_until" DATETIME DEFAULT NULL;
```

Comment:
Hmm, tests are failing on `DATETIME` for sqlite. I thought I had verified this locally since the short tests use sqlite 🤔

Reply:
I might've just misread the error last night- it looks like the sqlite tests were working, but we needed migrations specific for cockroach and postgres.